packages/mdc-select/mdc-select.scss

```diff
@@ -133,6 +133,9 @@
 // postcss-bem-linter: end
 
 .mdc-select__menu {
```

Comment:
Could you please make those changes on line 82 instead?

Reply:
see d4af29d